File: public/phase-2-20160620022203/JAVA/supergame.java
// Name : Supergame
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : number of boys
		// a :  number of rounds each boy wants to play
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=4;

		System.out.println(result);

	}
}

File: public/phase-2-20160620022203/C/supergame.c
// Name : Supergame
#include <stdio.h>
int main() {
	// input:
	// n : number of boys
	// a :  number of rounds each boy wants to play
	int n, a[10000], result=4, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}